### ISSUE TITLE
Surface Mesh Approxination, non documented variables

### ISSUE THREAD

**Issue by albert-github** [CGAL/cgal#8557]
When generating the documentation with `GENERATED_XML=YES` we get the warnings:
```
.../Surface_mesh_approximation/include/CGAL/Variational_shape_approximation.h:1531: warning: The following parameter of CGAL::Variational_shape_approximation::find_edges(const FT subdivision_ratio, const FT boundary_subdivision_ratio, const bool relative_to_chord, const bool with_dihedral_angle) is not documented:
  parameter 'boundary_subdivision_ratio'
.../Surface_mesh_approximation/include/CGAL/Variational_shape_approximation.h:1853: warning: The following parameter of CGAL::Variational_shape_approximation::subdivide_chord(const Boundary_chord_iterator &chord_begin, const Boundary_chord_iterator &chord_end, const FT subdivision_ratio, const FT boundary_subdivision_ratio, const bool relative_to_chord, const bool with_dihedral_angle) is not documented:
  parameter 'boundary_subdivision_ratio'
```

in the "normal"  documentation run we don't see these messages.

**Comment by sloriot** [comment]
Note that these are in private function so they shouldn't be reported. I fixed them nonetheless.

**Comment by albert-github** [comment]
Closed as completed by #8555
